flock_test.go

```diff
@@ -113,6 +155,28 @@ func (t *TestSuite) TestFlock_TryLockContext(c *C) {
 	c.Check(locked, Equals, false)
 }
 
+func (t *TestSuite) TestFlock_TryRLockContext(c *C) {
+	// happy path
+	ctx, cancel := context.WithCancel(context.Background())
+	locked, err := t.flock.TryRLockContext(ctx, time.Second)
+	c.Assert(err, IsNil)
+	c.Check(locked, Equals, true)
+
+	// context already canceled
+	cancel()
+	locked, err = flock.NewFlock(t.path).TryRLockContext(ctx, time.Second)
+	c.Assert(err, Equals, context.Canceled)
+	c.Check(locked, Equals, false)
+
+	// timeout
+	t.flock.Lock()
```

Comment:
Please call `.Unlock()` before locking. Not doing so breaks the tests on Windows.

Reply:
Done in ffd5ea4.